snuba/settings_test.py

```diff
@@ -1,3 +1,5 @@
+from coverage.data import os
```

Comment:
~What's the advantage of using this over just the standard library `os` module?~

```
>>> from coverage import data
>>> data.os
<module 'os' from '/Users/ted/.virtualenvs/snuba/bin/../lib/python3.7/os.py'>
```

Looks like it _is_ the standard library `os` module, please fix the indirect reference before merging:

```suggestion
import os
```